hasher-matcher-actioner/version.txt

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
Was debating whether we should use a < 1.0.0 version, do you have an opinion?

Reply:
I was close to do 0.0.1 lol, but also figured you may have a more strong opinion. Considering there seems to be users already with it in prod I think it is on a good state for it to be considered the first release tho.